### PR TITLE
Brutus responds to player position while at least one nurse is chasing

### LINKED ISSUE #292: Brutus mission, phase 2 reporter AI
Where all employees are able to report the player's position to Brutus once spotted.

### PR DESCRIPTION
**Pull request by fnnbrr**
Subject to change once the actual mission 3 prefab is in place, but the basic logic as it stands allows for Brutus to be alerted whenever Birdie is being chased.  This causes him to pathfind towards the player's last known location.

### CODE REVIEW

**Comment by fnnbrr** [comment]
@jurcaua I keep track of isPlayerSpotted in GameManager right now, but let me know when the prefab for mission 3 is ready & I'll keep track of it there instead.

**Comment by fnnbrr** [comment]
@jurcaua the Animator isn't rigged up for this yet, I figure one of us can do that later once Brutus is actually put in the maze chase scene and we're balancing his speed.

File: Assets/Scripts/NPCs/BrutusResponder.cs [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+            patroller = Utils.GetRequiredComponent<Patroller>(this);
+            waiter = Utils.GetRequiredComponent<Waiter>(this);
+            // animator = Utils.GetRequiredComponentInChildren<Animator>(this);
+            enemy = Utils.GetRequiredComponent<Enemy>(this);
+        }
+
+        public void Start()
+        {
+            chaser.OnSeePlayer += () => SetState(BrutusResponderStates.Chasing);
+            chaser.OnLosePlayer += () => nextState = BrutusResponderStates.Searching; SetState(BrutusResponderStates.Waiting);
```
Comment (jurcaua) [inline Assets/Scripts/NPCs/BrutusResponder.cs]: Loose `SetState(BrutusResponderStates.Waiting);` at the end there?

**Review by jurcaua — APPROVED** [review]
Small, probably just a formatting issue but otherwise looks great!!!! Let's get this bad boy in there 😈

File: Assets/Scripts/NPCs/BrutusResponder.cs (hunk 1 above)
Comment (fnnbrr) [reply Assets/Scripts/NPCs/BrutusResponder.cs]: Yep that's supposed to be there, let me fix the formatting though so it's actually legible

**Pull request merged**